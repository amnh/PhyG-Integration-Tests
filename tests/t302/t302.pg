--transform, inputting all character types, reducing dataset to 16 taxa, 
set(seed:1634561640)

read(rename: "woman_rename.txt")
read(aminoacid:"woman-wide_aa.fas")
read(fasta:"woman_12Saln.fasta", tcm:(1,1))
read(fasta:"woman_binary.fasta", tcm:(1,1))
read(prefasta:"woman_cox1aln.fasta")
read(fastc:"woman.fastc")
read(prefastc:"woman_cox1_spaced.fastc")
--woman_allchars.ss includes cc+, cc-, continuous, ?, [ ], /N 
read("woman_allchars.ss")
read("woman_step.ss")
read(include:"Include_woman16.txt")

read(newick:"../t92/t92.tre")
--transform(dynamicepsilon)
--select(unique:1)
--transform(staticApprox)
--swap(alternate, keep:1)
transform(dynamicepsilon)
swap()
report("t302.csv", data, overwrite)
report("t302.tre", newick, graphs, overwrite)