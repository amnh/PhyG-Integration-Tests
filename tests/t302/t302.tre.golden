(AlbanianTosk:2750.15,(Icelandic:5974.5,(ArmenianWestern:1892.25,((KolschColognian:7371.6,(Irish:1431.95,(Assamese:4682.05,Faroese:1534.6)HTU27:870.3000000000001)HTU25:1232.1)HTU26:1297.2,((Cornish:1560.3000000000002,OldPrussian:1468.7)HTU17:1689.6,(Manx:2946.6499999999996,(AncientIonicGreek:5673.05,(EasternFrisianSaterland:3161.45,(Scots:3539.5,(Danish:4293.45,(SanskritVedic:3519.1,Yiddish:3760.8)HTU22:3089.7)HTU23:1692.45)HTU20:1476.1)HTU21:1718.3)HTU18:1951.2)HTU24:1145.6)HTU30:1618.5)HTU19:1422.9)HTU28:844.3000000000001)HTU29:2553.15)HTU16[47892.6];
(AlbanianTosk:2411.15,(Icelandic:6229.0,(ArmenianWestern:2114.55,(((Cornish:1758.65,OldPrussian:1599.35)HTU17:2012.6,(Manx:3912.2999999999997,(AncientIonicGreek:4867.95,(EasternFrisianSaterland:2248.45,(Danish:2130.6,(SanskritVedic:3242.6,Yiddish:3474.8)HTU21:1653.05)HTU22:831.6)HTU20:1104.9)HTU18:2868.6499999999996)HTU23:1793.1)HTU29:1542.35,(Scots:2383.3,(KolschColognian:7611.1,(Irish:1947.95,(Assamese:4699.55,Faroese:1931.6)HTU26:577.3)HTU24:958.1)HTU25:986.6)HTU30:1455.7)HTU19:1765.1)HTU27:987.5)HTU28:2140.65)HTU16[47783.2];
(AlbanianTosk:2411.15,(Icelandic:6229.0,(ArmenianWestern:2114.55,(((Cornish:1758.65,OldPrussian:1599.35)HTU17:2012.6,(Manx:3912.2999999999997,(AncientIonicGreek:4867.95,(EasternFrisianSaterland:2248.45,(Danish:2130.6,(SanskritVedic:3242.6,Yiddish:3474.8)HTU21:1653.05)HTU22:831.6)HTU20:1104.9)HTU18:2868.6499999999996)HTU23:1793.1)HTU29:1542.35,(Scots:2383.3,(KolschColognian:7611.1,(Irish:1947.95,(Assamese:4699.55,Faroese:1931.6)HTU26:577.3)HTU24:958.1)HTU25:986.6)HTU30:1455.7)HTU19:1765.1)HTU27:987.5)HTU28:2140.65)HTU16[47783.2];
(AlbanianTosk:2411.15,(Icelandic:6229.0,(ArmenianWestern:2114.55,(((Cornish:1758.65,OldPrussian:1599.35)HTU17:2012.6,(Manx:3912.2999999999997,(AncientIonicGreek:4867.95,(EasternFrisianSaterland:2248.45,(Danish:2130.6,(SanskritVedic:3242.6,Yiddish:3474.8)HTU21:1653.05)HTU22:831.6)HTU20:1104.9)HTU18:2868.6499999999996)HTU23:1793.1)HTU29:1542.35,(Scots:2383.3,(KolschColognian:7611.1,(Irish:1947.95,(Assamese:4699.55,Faroese:1931.6)HTU26:577.3)HTU24:958.1)HTU25:986.6)HTU30:1455.7)HTU19:1765.1)HTU27:987.5)HTU28:2140.65)HTU16[47783.2];
(AlbanianTosk:2411.15,(Icelandic:6229.0,(ArmenianWestern:2114.55,(((Cornish:1758.65,OldPrussian:1599.35)HTU17:2012.6,(Manx:3912.2999999999997,(AncientIonicGreek:4867.95,(EasternFrisianSaterland:2248.45,(Danish:2130.6,(SanskritVedic:3242.6,Yiddish:3474.8)HTU21:1653.05)HTU22:831.6)HTU20:1104.9)HTU18:2868.6499999999996)HTU23:1793.1)HTU29:1542.35,(Scots:2383.3,(KolschColognian:7611.1,(Irish:1947.95,(Assamese:4699.55,Faroese:1931.6)HTU26:577.3)HTU24:958.1)HTU25:986.6)HTU30:1455.7)HTU19:1765.1)HTU27:987.5)HTU28:2140.65)HTU16[47783.2];
(AlbanianTosk:3866.65,(Icelandic:5978.5,(ArmenianWestern:1759.25,((KolschColognian:7783.1,(Irish:1907.45,(Faroese:1666.1,(Assamese:4645.5,Danish:2251.6499999999996)HTU26:763.1500000000001)HTU27:881.3000000000001)HTU24:1266.1)HTU25:1471.7,((Cornish:1476.3000000000002,OldPrussian:1416.7)HTU17:2002.6,(Manx:3122.1499999999996,(AncientIonicGreek:6370.55,(EasternFrisianSaterland:1891.45,(Scots:1742.8000000000002,(SanskritVedic:3387.3,Yiddish:3785.1)HTU22:1753.05)HTU20:744.8000000000001)HTU21:2654.3)HTU18:2413.2000000000003)HTU23:1265.1)HTU30:1713.0)HTU19:1826.4)HTU28:821.3000000000001)HTU29:3757.65)HTU16[47455.8];
(AlbanianTosk:3866.65,(Icelandic:5978.5,(ArmenianWestern:1759.25,((KolschColognian:7783.1,(Irish:1907.45,(Faroese:1666.1,(Assamese:4645.5,Danish:2251.6499999999996)HTU26:763.1500000000001)HTU27:881.3000000000001)HTU24:1266.1)HTU25:1471.7,((Cornish:1476.3000000000002,OldPrussian:1416.7)HTU17:2002.6,(Manx:3122.1499999999996,(AncientIonicGreek:6370.55,(EasternFrisianSaterland:1891.45,(Scots:1742.8000000000002,(SanskritVedic:3387.3,Yiddish:3785.1)HTU22:1753.05)HTU20:744.8000000000001)HTU21:2654.3)HTU18:2413.2000000000003)HTU23:1265.1)HTU30:1713.0)HTU19:1826.4)HTU28:821.3000000000001)HTU29:3757.65)HTU16[47455.8];
(AlbanianTosk:3866.65,(Icelandic:5978.5,(ArmenianWestern:1759.25,((KolschColognian:7783.1,(Irish:1907.45,(Faroese:1666.1,(Assamese:4645.5,Danish:2251.6499999999996)HTU26:763.1500000000001)HTU27:881.3000000000001)HTU24:1266.1)HTU25:1471.7,((Cornish:1476.3000000000002,OldPrussian:1416.7)HTU17:2002.6,(Manx:3122.1499999999996,(AncientIonicGreek:6370.55,(EasternFrisianSaterland:1891.45,(Scots:1742.8000000000002,(SanskritVedic:3387.3,Yiddish:3785.1)HTU22:1753.05)HTU20:744.8000000000001)HTU21:2654.3)HTU18:2413.2000000000003)HTU23:1265.1)HTU30:1713.0)HTU19:1826.4)HTU28:821.3000000000001)HTU29:3757.65)HTU16[47455.8];
(AlbanianTosk:4049.15,(Icelandic:6227.0,(ArmenianWestern:2127.55,(((Cornish:1765.3000000000002,OldPrussian:1543.7)HTU17:2287.6,(Manx:3446.6499999999996,(AncientIonicGreek:6469.1,(EasternFrisianSaterland:1896.9,(SanskritVedic:3243.6,Yiddish:3411.8)HTU21:1402.6)HTU20:3031.2999999999997)HTU18:2862.1499999999996)HTU22:1576.6)HTU29:1757.5,(Scots:2662.8,(KolschColognian:8048.1,(Irish:2210.45,(Faroese:1791.1,(Assamese:4646.5,Danish:2405.6499999999996)HTU25:668.6500000000001)HTU26:780.3000000000001)HTU23:1158.1)HTU24:1130.6)HTU30:1557.2)HTU19:2087.1)HTU27:965.5)HTU28:4006.65)HTU16[47401.8];
(AlbanianTosk:4049.15,(Icelandic:6227.0,(ArmenianWestern:2127.55,(((Cornish:1765.3000000000002,OldPrussian:1543.7)HTU17:2287.6,(Manx:3446.6499999999996,(AncientIonicGreek:6469.1,(EasternFrisianSaterland:1896.9,(SanskritVedic:3243.6,Yiddish:3411.8)HTU21:1402.6)HTU20:3031.2999999999997)HTU18:2862.1499999999996)HTU22:1576.6)HTU29:1757.5,(Scots:2662.8,(KolschColognian:8048.1,(Irish:2210.45,(Faroese:1791.1,(Assamese:4646.5,Danish:2405.6499999999996)HTU25:668.6500000000001)HTU26:780.3000000000001)HTU23:1158.1)HTU24:1130.6)HTU30:1557.2)HTU19:2087.1)HTU27:965.5)HTU28:4006.65)HTU16[47401.8];
